src/docs/sphinx/dev_guide/gpu_porting.rst

```diff
@@ -0,0 +1,302 @@
+.. ## Copyright (c) 2017-2022, Lawrence Livermore National Security,con LLC and
```

Comment:
What is "Security,con LLC" ?

Reply:
That's a typo. ',con' should be removed.

Reply:
```suggestion
.. ## Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
```